.editorconfig

```
@@ -0,0 +1,8 @@
+root = true
+
+[*.{rb}]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```

A final newline subtly improves diffs once adopted in a project.

```suggestion
insert_final_newline = true
```